OrbitGl/App.cpp

```diff
@@ -111,21 +110,6 @@ void OrbitApp::SetCommandLineArguments(const std::vector<std::string>& a_Args) {
 
   for (const std::string& arg : a_Args) {
     if (absl::StrContains(arg, "gamelet:")) {
```

Comment:
We should support the old syntax for a while and output a deprecation message.  We'll need to change the docs for the new flag.

Reply:
+1